feat: Adds `protected_hours` and `time_zone_id` to `mongodbatlas_maintenance_window` (#3195)
mongodb/terraform-provider-mongodbatlas: examples/mongodbatlas_maintenance_window/README.md

# MongoDB Atlas Provider - Configure Maintenance Window

> This example shows how you can configure maintenance windows for your Atlas project in Terraform.

Reviewer: ```suggestion
This example demonstrates how to configure maintenance windows for your Atlas project in Terraform.
```

Reviewer 2: Thank you applied in 3bd6f5cdcf4cc8643dfa8c6e1f58a435e7fe7363

> Variables required to be set:

Reviewer: ```suggestion
Required variables to set:
```

Reviewer 2: Thank you applied in 3bd6f5cdcf4cc8643dfa8c6e1f58a435e7fe7363

- `public_key`: Atlas public key
- `private_key`: Atlas private key

> - `org_id`: Organization ID where the project and cluster will be created.

Reviewer: ```suggestion
- `org_id`: Unique 24-hexadecimal digit string that identifies the organization that contains the project and cluster.
```

Reviewer 2: Thank you applied in 3bd6f5cdcf4cc8643dfa8c6e1f58a435e7fe7363
